swagger.yaml

```diff
@@ -844,4 +957,37 @@ components:
         lastUpdatedAt:
           type: string
           format: date-time
-          description: Date and time when the software entry was last updated
\ No newline at end of file
+          description: Date and time when the software entry was last updated
+        
+      Questionnaire:
```

Comment:
Few thoughts about potential missing properties:
-Do we need some kind of passingScore attribute? 
-How will we record who has passed a questionnaire? Perhaps it makes more sense to record the questionnaires passed on the user entity Iiro is working on but what do you think? We might also want to record the score for a user, rather than just if they passed or failed.
-It would be very nice if the Questionnaire could also have some kind of tags for example this would allow us to show if it id targeted to a developer or a designer, or perhaps if its related to security vs company policy etc.